cmd/create/cmd.go

```diff
@@ -61,6 +62,7 @@ func init() {
 	Cmd.AddCommand(dnsdomains.Cmd)
 	Cmd.AddCommand(autoscaler.Cmd)
 	Cmd.AddCommand(kubeletconfig.Cmd)
+	Cmd.AddCommand(externalauthproviders.Cmd)
```

Comment:
Important, this command needs a flag to be hidden:

```
$ rosa create
Create a resource from stdin

Usage:
  rosa create [command]

Aliases:
  create, add

Available Commands:
  account-roles           Create account-wide IAM roles before creating your cluster.
  admin                   Creates an admin user to login to the cluster
  autoscaler              Create an autoscaler for a cluster
  cluster                 Create cluster
  dns-domain              Create DNS Domain.
  external-auth-providers Create external authentication providers for a cluster.
...
```

There is a similar syntax as the hidden flag for an attribute.

Reply:
Another minor, perhaps in singular is more accurate `external-auth-provider`.

Reply:
@lucasponce It's now hidden

Reply:
Changed to singular and hided the cmd.